### ISSUE TITLE
VS code hangs on infinite loop on cpp

### ISSUE THREAD

**Issue by Spartan9151** [microsoft/vscode#117810]
We have written the needed data into your clipboard because it was too large to s
Issue Type: <b>Bug</b>

Whenever my program gets into an infinite loop, vs code hangs and all functionality becomes unavailable.
I have to manually cloase the process.
Please do this bugfix

VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:48:04.245Z)
OS version: Windows_NT x64 10.0.19041

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz (12 x 2208)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.85GB (2.00GB free)|
|Process Argv|--crash-reporter-id 5b3ce09c-9525-4074-8ef4-acc17139c067|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
code-runner|for|0.11.2
anaconda-extension-pack|ms-|1.0.1
python|ms-|2021.2.582707922
jupyter|ms-|2021.2.603412351
cpptools|ms-|1.2.2
java|red|0.75.0
vscode-yaml|red|0.15.0
vscodeintellicode|Vis|1.2.11
vscode-java-debug|vsc|0.31.0
vscode-java-dependency|vsc|0.18.0
vscode-java-pack|vsc|0.12.1
vscode-java-test|vsc|0.28.0
vscode-maven|vsc|0.27.1

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspyt653:30253241
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry914:30256638
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
dockersubset:30264244
pythontb:30260843

```

</details>

end. Please paste.

**Comment by gjsjohnmurray** [comment]
Probably #74620 which shows as fixed in the upcoming 1.54.

Either wait for the release, due out in the first week of March, or try Insiders which can be installed alongside Stable.

**Comment by deepak1556** [comment]
Can you check if the issue is present in insiders https://code.visualstudio.com/insiders as mentioned above.
